docs/deployment.md

```diff
@@ -9,27 +9,30 @@ Once this is done, the automatic deployment to staging begins. This deployment u
 For staging it is good enough -- we can allow for some small downtime in favor of lowering the complexity
 of the set up.
 
-TBD Production.
+Currently same strategy is used for production, but this will be changed in the future.
 
 ## Service Account
 
-In order to deploy the service (apply the Kustomize output) we need to have a dedicated Service Account
+In order to deploy the service (via Helm) we need to have a dedicated Service Account
 which can be used to authenticate against the target cluster. We need such a Service Account for every
 target environment.
 
-```
+```bash
 oc create sa sbomer-deployer
 ```
 
+NOTE: The _ServiceAccount_ name may differ between [environments](environments.md).
+
 ### Required RoleBindings
 
-By default no additional RoleBindings are required.
+A *ClusterRole* `admin` should be applied to this *ServiceAccount* for a given namespace.
```

Comment:
Can you describe also how to apply a ClusterRole to a ServiceAccount.

Reply:
I'm a bit unsure how to deal with his section in general. The SA and RoleBindings are already created on the target (stage/prod) envs. These won't change. For development environment we don't need these.

Thoughts?